test/test_nn.py

```diff
@@ -7847,6 +7848,11 @@ def test_softmin(self):
         self.assertEqual(F.softmin(x, 1), F.softmax(-x, 1))
         self.assertEqual(F.softmin(x, 0), F.softmax(-x, 0))
 
+    def test_log_softmax(self):
+        x_small = torch.ones(1, 2, dtype=torch.float32)
+        x_big = x_small * 1e16
```

Comment:
softmax and logsoftmax should be invariant uner addition, but not multiplication, right? (multilicative factor is inverse temperature, right?)

should this line read  `x_big = x_small + 1e16`?

Reply:
oops..., this test works in the special case, but it's a mistake. I've written a new commit.